docs/cli/discount-code-generator.md

```diff
@@ -0,0 +1,153 @@
+# Discount Code Generator
+
+Generate unique discount codes to be imported to the [commercetools platform](https://dev.commercetools.com/). See usage below
+
+## Usage
+`npm install @commercetools/discount-code-generator --global`
+
+### CLI
+```
+Usage: discountCodeGenerator [options]
+
+Options:
+  --help, -h                 Show help text.                                                     [boolean]
```

Comment:
Hello @wizzy25 .. can you add that `--logLevel` param also here to the documentation?